enterprise/jakarta.transformer/src/org/netbeans/modules/fish/payara/jakarta/transformer/TransformerConstants.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.fish.payara.jakarta.transformer;
+
+/**
+ *
+ * @author alfon
+ */
+public class TransformerConstants {
+    
+    public static final String PAYARA_TRANSFORMER_GROUP_ID = "fish.payara.transformer";
+    
+    public static final String PAYARA_TRANSFORMER_ARTIFACT_ID = "fish.payara.transformer.maven";
+    
+    public static final String PAYARA_TRANSFORMER_VERSION = "0.2.12";
+    
+    public static final String PAYARA_TRANSFORMER_GOAL = "run";
+           
+    public static final String SELECTED_SOURCE_PROPERTY_NAME = "selectedSource";
+    
+    public static final String SELECTED_TARGET_PROPERTY_NAME = "selectedTarget";
+    
+    public static final String PROJECT_NAME = "payara-sample";
```

Comment:
This will place all transformed files into the `payara-sample` folder, which is a bit confusing. Would you like to change this to something like `jakartaee10-sample`? Or `jakartaee-transformed-sample`? Or even make it configurable in the dialog?

Reply:
I already changed the name from payara-sample to jakartaee-transformed-sample. I think this is better because this is used just to save result of transformation when you select a target folder from the wizard.